webpack/assets/javascripts/react_app/components/BreadcrumbBar/BreadcrumbBar.js

```diff
@@ -12,9 +12,13 @@ class BreadcrumbBar extends React.Component {
       loadSwitcherResourcesByResource,
       currentPage,
       resourceUrl,
+      isCorrectList,
     } = this.props;
-
-    if (!currentPage || resourceUrl !== resource.resourceUrl) {
+    if (
+      !currentPage ||
+      resourceUrl !== resource.resourceUrl ||
+      !isCorrectList
```

Comment:
I'm not sure using `isCorrectList` as a prop would be the best choice.

See `resource.switcherItemUrl`, it contain the template for how the urls should be built.
For example:
```js
// edit host
resource.switcherItemUrl = '/hosts/:id/edit';
// show host
resource.switcherItemUrl = '/hosts/:id/';
```

Could you use this template and make sure the current browser location match the template?

Reply:
Done